recipes/rnaprot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "RNAProt" %}
+{% set version = "0.3" %}
+{% set sha256 = "18ffa9d569c63e3fc5d7a31427c7569d6021c67587a6838aea4ff3af7f0780e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/BackofenLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  noarch: python
+  script: $PYTHON -m pip install . --ignore-installed --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pytorch-gpu 1.7.1  # [cuda_compiler_version != "None"]
+    - pytorch-cpu 1.7.1  # [cuda_compiler_version == "None"]
```

Comment:
Does this work?

Reply:
I don't know, how can I test this ? I guess I would have to "conda install" them to see ...

Reply:
You mean like this:
`conda install -c https://149193-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages rnaprot ` ? Not sure how this works ...

Reply:
Yes :+1:

Reply:
Doesn't work no:
```
$ conda install -c https://149193-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages rnaprot
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: / 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                    

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions


```

Reply:
Unfortunately not very informative :/

Reply:
I tried on two computers (with and without CUDA GPU), same error.

Reply:
It works for me, are you in a fresh conda env?

Unfortunately it is installing `pytorch-gpu` on my cpu-only machine.

Reply:
Yes I created one for testing:
`conda create -n testenv python=3.8`

Reply:
Conda version is 4.10.0

Reply:
We could try this from the other recipe (not sure if undefined is also needed):
```
  - pytorch-cpu   # [cuda_compiler_version in (undefined, 'None')]
  - pytorch-gpu   # [cuda_compiler_version not in (undefined, 'None')]
```